rust-runtime/aws-smithy-runtime/src/test_util/capture_test_logs.rs

```diff
@@ -12,7 +12,37 @@ use tracing_subscriber::fmt::TestWriter;
 
 /// A guard that resets log capturing upon being dropped.
 #[derive(Debug)]
-pub struct LogCaptureGuard(#[allow(dead_code)] DefaultGuard);
+pub struct LogCaptureGuard(#[allow(dead_code)] Option<DefaultGuard>);
+
+/// Enables output of test logs to stdout.
+///
+/// The `VERBOSE_TEST_LOGS` environment variable acts as a
+/// tracing_subscriber fmt env filter. You can give it full env filter
+/// expressions, or just simply give it a log level (e.g., tracing, debug, info, etc).
+/// Setting it to "1" or "true" will enable trace logging.
+#[must_use]
+pub fn show_test_logs() -> LogCaptureGuard {
+    let (mut writer, _rx) = Tee::stdout();
+    let env_var = env::var("VERBOSE_TEST_LOGS").ok();
+    let env_filter = match env_var.as_deref() {
+        Some("true") | Some("1") => Some("trace"),
+        Some(filter) => Some(filter),
+        None => None,
+    };
```

Comment:
I think we should add a way to loudly show test logs without having to set an env var for convenience (e.g. when running tests w/ intellij)

Could make a `loud` argument to the variable, or add a boolean arg to the function?

Or keep `show_test_logs` as the loud one?